src/recap.js

```diff
@@ -203,7 +203,7 @@ function Recap() {
           console.info(`RECAP: Successfully uploaded PDF: 'Success' ` +
             `with processing queue id of ${result.id}`);
           cb(result || null);
-          destroyTabStorage(cb.tab.id);
+          updateTabStorage({ [cb.tab.id]: { ['pdf_blob']: null } });
```

Comment:
I thought I'd see something explaining this, but I don't think I did. What's going on here?

Reply:
The current implementation of the extension fails to identify pages if a user clicks the back button after a PDF file is uploaded from Appellate because pages like the **docket report** and the **attachment page** try to get the `pacer_case_id` from the tab storage, but the extension destroys the whole object after the upload request completed. 

I assume that the extension destroys the tab storage because it has an attribute that stores the PDF document (to share it with the background worker), but the update operation can also remove the PDF data without destroying the whole object.

Reply:
Using the update method instead of the destroy function fixes the issue with the pages from Appellate and also removes the PDF data from the tab storage.